sdk/identity/Azure.Identity/tests/ImdsManagedIdentitySourceTests.cs

```diff
@@ -92,7 +93,8 @@ public async Task DefaultAzureCredentialUsesFirstRequestBehaviorUntilFirstRespon
                 ExcludeVisualStudioCodeCredential = true,
                 ExcludeVisualStudioCredential = true,
                 ExcludeWorkloadIdentityCredential = true,
-                Transport = mockTransport
+                Transport = mockTransport,
+                IsForceRefreshEnabled = true
```

Comment:
Do we have tests with this bool set to false (presumably that's the default value if not set)?

Reply:
Yes, there are some - but many of these mock transport tests require this configuration to run more than 1 test in the same test run properly. Otherwise, we get token responses from cache and cannot observed the request to the authority, which these tests validate.